Civi/Core/Container.php

```diff
@@ -203,8 +203,12 @@ public function createContainer() {
       'Civi\Token\TokenCompatSubscriber',
       array()
     ))->addTag('kernel.event_subscriber');
+    $container->setDefinition("crm_mailing_action_tokens", new Definition(
```

Comment:
Yup, the intent is to address that readability issue by moving the declarations to smaller/separate files. Outside scope of the current PR, though.